foodnet/membership/models.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+
+
+class UserProfile(models.Model):
+    MALE = 'm'
+    FEMALE = 'f'
+    SEX_CHOICES = (
+        (FEMALE, 'female'),
+        (MALE, 'male')
+    )
+    
+    user = models.OneToOneField(User)
+    middlename = models.CharField(max_length=255)
+    adr1 = models.CharField(max_length=255)
+    adr2 = models.CharField(max_length=255)
+    streetno = models.CharField(max_length=255)
+    floor = models.CharField(max_length=255)
+    door = models.CharField(max_length=255)
```

Comment:
You could probably do `adr1`, `adr2`, `streetno`, `floor` and `door` in one field called `address`. There is no significant value in breaking them up I would say.

Also, I would highly recommend to refrain from using shortened attribute names as `adr1` and `streetno`. Just type it all out. `profile.street_number` isn't that bad :)